source/MaterialXTest/MaterialXCore/Look.cpp

```diff
@@ -98,24 +101,90 @@ TEST_CASE("LookGroup", "[look]")
     std::vector<mx::LookGroupPtr> lookGroups = doc->getLookGroups();
     REQUIRE(lookGroups.size() == 1);
 
+    mx::StringMap mergeMap;
+
     const std::string looks = "look1,look2,look3,look4,look5";
-    mx::StringVec looksVec = mx::splitString(looks, ",");
+    mx::StringVec looksVec = mx::splitString(looks, mx::ARRAY_VALID_SEPARATORS);
```

Comment:
Note: We should be using ARRAY_VALID_SEPARATORS which allows for "," or ", " as the separator.